util/mod_translation_updater.py

```diff
@@ -121,17 +120,43 @@ def main():
 
 # Group 2 will be the string, groups 1 and 3 will be the delimiters (" or ')
 # See https://stackoverflow.com/questions/46967465/regex-match-text-in-either-single-or-double-quote
-pattern_lua_quoted = re.compile(r'[\.=^\t,{\(\s]N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
+pattern_lua_quoted = re.compile(
+	r'(?:^|[\.=^\t,{\(\s])' # Look for beginning of file or anything that isn't a function identifier
```

Comment:
```suggestion
	r'(?:^|[\.=^\t,{\(\s])' # Look for beginning of line or anything that isn't a function identifier
```

also: what's that `^` doing in the bracket capture? that's not an operator in Lua, is it?
also (2): `\s` already includes `\t`